bacon.toml

```diff
@@ -0,0 +1,11 @@
+default_job = "check"
```

Comment:
Is this tied into crates.io? Or just a tool you started using?

Reply:
Just a tool I used for development, and the config showed up as "dirty", blocking publishing.

Reply:
Gotcha. Definitely a nit, but since other bacon users might have different config this strikes me as something that could go `.gitignore`. I'd even suggest global gitignore rather than project since it is developer-specific config (i.e. not everyone might be using bacon).